src/requirements/Sound/SoundRequirement.tsx

```diff
@@ -0,0 +1,157 @@
+import { Img } from "@chakra-ui/react"
+import Link from "components/common/Link"
+import { RequirementComponentProps } from "requirements"
+import Requirement from "requirements/common/Requirement"
+import slugify from "slugify"
+import useSWRImmutable from "swr/immutable"
+
+const SoundRequirement = ({ requirement, ...rest }: RequirementComponentProps) => {
+  const { data: artistsData, isValidating: artistsLoading } = useSWRImmutable(
+    `/api/sound-artists?searchQuery=${requirement.data?.id}`
+  )
+
+  const { data: songsData, isValidating: songsLoading } = useSWRImmutable(
+    `/api/sound-songs?id=${artistsData?.map((artist) => artist[0].id)}`
+  )
+
+  const songImageUrl = songsData
+    ?.filter((title) => title[0].title == requirement.data.title)
+    .map((song) => song[0].image)
+
+  return (
+    <Requirement
+      image={(() => {
+        switch (requirement.type) {
+          case "SOUND_ARTIST":
+            return <Img src="/requirementLogos/sound.png" />
+
+          case "SOUND_ARTIST_BACKED":
+            return artistsData?.map((artist) => artist[0].image) ? (
+              <Img src={artistsData?.map((artist) => artist[0].image)} />
+            ) : (
+              <Img src="/requirementLogos/sound.png" />
+            )
+
+          case "SOUND_COLLECTED":
+            return songsData?.map((song) => song[0].image) ? (
+              <Img src={songImageUrl} />
+            ) : (
+              <Img src="/requirementLogos/sound.png" />
+            )
+          case "SOUND_TOP_COLLECTOR":
+            return artistsData?.map((artist) => artist[0].image) ? (
+              <Img src={artistsData?.map((artist) => artist[0].image)} />
+            ) : (
+              <Img src="/requirementLogos/sound.png" />
```

Comment:
should just have a `default: return <Img src="/requirementLogos/sound.png" />` at the end and conditional returns in earlier cases